cosmos/providers/dbt/render.py

```diff
@@ -96,7 +105,8 @@ def render_project(
         # make the group
         model_group = Group(
             id=model_name,
-            entities=[run_task, test_task],
+            entities=[docs_task, run_task, test_task],
+            #entities=[run_task]
```

Comment:
`dbt-ol` bug breaks the lineage graph after `dbt-ol test` is run, but testing with just `dbt-ol run` worked.